src/user-guide/task-implementation/task-run-modes.rst

```diff
@@ -0,0 +1,112 @@
+.. _task-run-modes:
+
+Task Run Modes
+==============
+
+.. versionadded:: 8.4.0
+
+   * Before 8.4.0 run modes could only be set for the workflow, not
+     for individual tasks.
+   * Skip mode.
+
+
+As well as the default task implementation ("live" mode)
+other implementations can be selected:
```

Comment:
I think it would be more helpful if this whole page was framed as about skip mode. For the user, I don't think it's useful to talk about "multiple" task run modes; what is important to them is they can set tasks to run in skip mode.